guides/alerting/alerting.md

```diff
@@ -33,19 +38,64 @@ SMTP_USER=user
 SMTP_PASSWORD=password
 ```
 
+The other way of configuring alerting is via the Web interface or
+through the API. To be able to use this variant, _all_ of the
+aforementioned env-vars should be left undefined when starting
+Trento. Then, configuring the alerting values could be done in a
+self-explanatory manner from Settings menu in Trento web
+console. There is also an OpenAPI spec for the API.
+
+Currently **SMTP** is the **only supported delivery mechanism** for
+alert notifications.
+
 ## Enabling Alerting at a later stage
-If your current Trento installation has Alerting disabled, you can enable it by upgrading the helm deployment.
+
+If no env-vars are in place, then you can freely change alerting
+configuration via WebUI and/or RESTful API at any time without needing
+to restart.
+
+If your current Trento installation has alerting env-vars in-place,
+then in order to change them you have to restart Trento with the new
+values. If using Helm, for example, you can change alerting env-vars
+by upgrading the deployment:
 
 ```
 helm upgrade
-   --install <THE_DEPLOYMENT> 
+   --install <THE_DEPLOYMENT>
    --set trento-web.adminUser.password=<ADMIN_PASSWORD>
    --set-file trento-runner.privateKey=<PRIVATE_SSH_KEY>
    --set trento-web.alerting.enabled=true
-   --set trento-web.alerting.smtpServer=<SMTP_SERVER> 
+   --set trento-web.alerting.smtpServer=<SMTP_SERVER>
    --set trento-web.alerting.smtpPort=<SMTP_PORT>
    --set trento-web.alerting.smtpUser=<SMTP_USER>
    --set trento-web.alerting.smtpPassword=<SMTP_PASSWORD>
-   --set trento-web.alerting.sender=<ALERT_SENDER> 
+   --set trento-web.alerting.sender=<ALERT_SENDER>
    --set trento-web.alerting.recipient=<ALERT_RECIPIENT>
 ```
+
+## Local development and testing
+
+Our `docker-compose.yaml` file contains a profile providing a Mailpit
```

Comment:
What do you think about: 
The provided docker-compose.yaml file includes a profile that launches a Mailpit SMTP server for local testing.

To start Mailpit, run:

.....

Access Mailpit’s web interface at: http://localhost:8025/